misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/launchrequest/LauncherImpl.java

```diff
@@ -75,7 +75,7 @@
 
         // TODO: validate file path
         ArrayList<String> command = new ArrayList<>(ballerinaExec);
-        boolean debugTests = args.get("debugTests") != null && Boolean.parseBoolean((String) args.get("debugTests"));
+        boolean debugTests = args.get("debugTests") != null && (boolean) args.get("debugTests");
```

Comment:
Please avoid this unsafe casting of client args (`object` -> `boolean`)

Reply:
Fixed